package.json

```diff
@@ -11,8 +11,7 @@
     "start": "next start",
     "test": "jest --watch",
     "test:noWatch": "jest",
-    "ci-full": "npm run lint && jest --ci && WITH_DEBUG=true npm run build && npm run cy:ci",
-    "ci": "npm run lint && jest --ci",
+    "ci": "npm run lint && jest --ci && WITH_DEBUG=true npm run build && npm run cy:ci",
```

Comment:
inac tu treba `npm run build` ? Lebo `npm run cy:ci` uz ma build v sebe

Reply:
`cy:ci` vola `start-server-and-test` a podla docs robi:
1. `npm start`
2. caka kym je na URL http://localhost:3000/ nieco spustene
3. `npm run cy:ci`

Alebo ten build je tam nejako defaultne?

Ja som totiz uz davnejsie menil v `cy:ci` aby sa nemustal `dev` ale `start`, ide to totiz o trochu rychlejsie (v Travise). A preto treba aj `WITH_DEBUG=true` pri buildovani aby sa zobrazil `Debug` component.

Reply:
Ok, sorry myslel som ze `npm start` defaultne najprv buildne a potom nastartuje server, ale nie je to tak, treba buildnut. Takze takto je to spravne.